synapse/cores/common.py

```diff
@@ -2264,82 +2248,6 @@ def _primToProps(self, form, item):
 
         return props
 
-    def addTufoEvents(self, form, propss):
```

Comment:
:+1: There is no longer a need for this method, as you can get the same behavior by passing `*` to the valu of a call to `formTufoByProp`